modules/svg/data.js

```diff
@@ -157,7 +157,7 @@ export function svgData(projection, context, dispatch) {
 
 
     function isPolygon(d) {
-        return d.geometry.type === 'Polygon' || d.geometry.type === 'MultiPolygon';
+        return d.geometry.type === 'Polygon' || d.geometry.type === 'MultiPolygon' || d.geometry.type === 'GeometryCollection';
```

Comment:
This will assume all GeometryCollections should be styled like areas (with the inner shading).  Might I suggest a boolean expression like this to test the types in the geometries property:
```
|| (d.geometry.type === 'GeometryCollection' && d.geometry.geometries.every(function(g) {
+            return g.type === 'Polygon' || g.type === 'MultiPolygon';
+        }))
```

Reply:
![image](https://user-images.githubusercontent.com/2600932/99863537-97500c00-2b6c-11eb-9cec-a95f6d3c7132.png)
vs
![image](https://user-images.githubusercontent.com/2600932/99863557-afc02680-2b6c-11eb-8dc6-4fca9730058c.png)